spec/dor/publishable_spec.rb

```diff
@@ -345,7 +345,7 @@ class ItemizableItem < ActiveFedora::Base
   describe 'publish remotely' do
     before(:each) do
       Dor::Config.push! {|config| config.dor_services.url 'https://lyberservices-test.stanford.edu/dor'}
-      allow_any_instance_of(RestClient::Resource).to receive(:post)
+      stub_request(:any, 'https://lyberservices-test.stanford.edu/dor/v1/objects/druid:ab123cd4567/publish')
     end
```

Comment:
Maybe the spec could expect that the client used is `RestClient`?  If it is changed, the specs will fail and point the way to changing any code that depends specifically the `RestClient`.